ideas/88-dapp-experience.md

```diff
@@ -0,0 +1,87 @@
+## Preamble
+
+    Idea: 88
+    Title: DApp Experience
+    Status: Draft
+    Created: 2018-03-12
+
+## Summary
+
+Interacting with DApps is in the very core of Status. In fact, the shortest description we have for Status is *Mobile Ethereum OS (or Browser)*. 
+
+The problem is that new users do not really get a chance to see this aspect of Status, simply because there are very few DApps accessible directly through Status. The only way to find new DApps is on external web sites or search engines. Having to use an external app to access a core feature of our app is bad UX. Furthermore, users who are completely new to these technologies and generally unfamiliar with the concept of DApps might not even think to look for them and would only use Status as a messenger or possibly a wallet.
+
+Briefly put, the minimum we need to provide is a nice selection of DApps so that our users could use our Browser feature.
+
+We had two brainstorming sessions during the Bangkok offsite. The notes from there are on the link below, please do not hesitate to comment:
+
+ https://docs.google.com/document/d/11MRsAO8GdIKVB0bk-osld-cYMMWDOVxB012QHPkQ6_k/edit?usp=sharing
+
+## Swarm Participants
+
+- Lead Contributor: @goranjovic
+- Testing & Evaluation: @asemiankevich 
+- Contributor: @lukaszfryc 
+- Contributor: <!-- @username -->
+- PM: @rachelhamlin
+- UX (if relevant): <!-- @username -->
+
+## Product Overview
+
+Objectives:
+
+1. Provide a definition of decentralized apps
```

Comment:
Is it also about an onboarding process for our users? It maybe necessary to describe what is a dapp inside Status. 

Of course we also need a good description, logo, etc. (assets) for each listed dapp.

Reply:
It might be good to have a special introductory dapp that demonstrates the basic dapp concepts upon account creation. I'm just not sure exactly what that dapp should do exactly and all my thoughts around it have somehow been about that dreaded Office Paperclip

Reply:
Poor Clippy. There was an idea in BKK about an incentivized onboarding dapp, that could teach users about Status and reward completion with SNT.

How about we keep it to description and assets for each dapp? We can build out the discovery experience later, but the early audience should have some grasp on dapps, and later audiences shouldn’t need to think about it. If we want to externalise our decision-making, it could be part of our comms strategy.